src/pages/Login/Login.js

```diff
@@ -1,7 +1,97 @@
-import React from 'react';
+import React, { useState } from 'react';
+import './Login.scss';
+import Nav from '../../components/Nav/Nav';
+import Footer from '../../components/Footer/Footer';
 
 function Login() {
-  return <div>Login</div>;
+  let [loginEmail, setLoginEmail] = useState('');
+  let [loginPw, setLoginPw] = useState('');
+  let loginValid = loginEmail.includes('@') && loginPw.length > 7;
+  let validInformE = loginEmail.includes('@');
+  let validInformPw = loginPw.length > 7;
```

Comment:
```suggestion
  const validInformEmail = loginEmail.includes('@');
  const validInformPw = loginPw.length > 7;
```
유효성 검사하는 조건을 state로 관리하지 않고 변수로만 선언해도 충분히 가능할 것 같네요, 참고해서 수정해 보세요 :)
그리고 변수명 조금 더 디테일하게 정해주세요! E 보다는 Email이라고 명확하게 의미를 드러나게 해주세요!